airbyte-webapp/src/components/connectorBuilder/Builder/BuilderCard.tsx

```diff
@@ -33,7 +52,139 @@ export const BuilderCard: React.FC<React.PropsWithChildren<BuilderCardProps>> =
           {toggleConfig.label}
         </div>
       )}
+      {copyConfig && streams.length > 1 && (
+        <div className={styles.copyButtonContainer}>
+          {get(streams[copyConfig.currentStreamIndex], copyConfig.path) && (
```

Comment:
nit: swap the order of the copy to/from buttons, so that the copy from button stays in the same place regardless of if the card is toggled on or off